component/background/RoomService.tsx

```diff
@@ -11,11 +12,16 @@ const config = {
 };
 firebase.initializeApp(config);
 
-class RoomService {
+const allRooms = firebase.database().ref("rooms");
+const allUsers = firebase.database().ref("users");
+
+const MESSAGE = "message";
+const USER = "user";
```

Comment:
Before compile time, we should have code that clarifies this by being within its proper scope.